frontend/src/services/settings.ts

```diff
@@ -11,7 +11,7 @@ export type Settings = {
 };
 
 export const DEFAULT_SETTINGS: Settings = {
-  LLM_MODEL: "openai/gpt-4o",
+  LLM_MODEL: "claude-3-5-sonnet-20241022",
```

Comment:
Should this be:
```suggestion
  LLM_MODEL: "anthropic/claude-3-5-sonnet-20241022",
```

Reply:
Both should work in the backend, but since it was with provider prefix here, maybe we can have it with `anthropic` indeed

Reply:
seems to work as is...